### ISSUE TITLE
[UX] Confirmation on remove

### ISSUE THREAD

**Issue by f1xmAn** [gpouilloux/gnome-shell-extension-docker#29]
Please provide confirmation dialog on container removal operation. I've just accidentally deleted my container while wanted to start it.

**Comment by f1xmAn** [comment]
God, please! I did it again!

**Comment by Alan-M-Thomaz** [comment]
Yeah, just done this on my local development database....,  now i'm  making a 2 hour dump from QA, cause i need some data to work....

**Comment by alessandrodolci** [comment]
@gpouilloux I'm working on this, I will open a PR soon, hope it's ok for you!

**Comment by gpouilloux** [comment]
Thanks @alessandrodolci , that is an useful feature.
